frontend/src/components/PropertyCard.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import {
+    Card,
+    Image,
+    CardContent,
+    TitleContainer,
+    CardTitle,
+    DetailsButton,
+    DetailsButtonVectors,
+    Vector,
+    AddressContainer,
+    AddressContainerVectors,
+    AddressContainerTitle,
+    KpiContainer,
+    ValueContainer1,
+    ValueDescription1,
+    ValueField1,
+    Value1
+} from "../styles/PropertyCard.style";
+
+const PropertyCard = () => {
+    return (
+        <Card>
+            <Image
+                alt=""
+                src="https://static.overlay-tech.com/assets/11b6e432-01e3-49a6-af28-ce04e08b8de6.png"
+            />
+            <CardContent>
+                <TitleContainer>
+                    <CardTitle>Write a title here</CardTitle>
+                    <DetailsButton>
+                        <DetailsButtonVectors>
+                            <Vector
+                                alt=""
+                                src="https://static.overlay-tech.com/assets/17426534-e0c5-4d48-b293-6bc7c6a040dd.svg"
+                            />
+                        </DetailsButtonVectors>
+                    </DetailsButton>
+                </TitleContainer>
```

Comment:
Maybe move these containers to an own file? And move the styling to each if these files.